src/brotli/build.py

```diff
@@ -2,6 +2,7 @@
 import sys
 
 from cffi import FFI
+
```

Comment:
Eh, I don't care very much either. That's why I didn't flag the other thing that was very obviously a change from isort. =P